specification/resourcegraph/resource-manager/Microsoft.ResourceGraph/preview/2018-09-01-preview/resourcegraph.json

```diff
@@ -411,6 +495,142 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceChangeData"
+          }
+        },
+        "$skipToken": {
+          "description": "Skip token that encodes the skip information while executing the current request"
+        }
+      }
+    },
+    "ResourceChangeData": {
+      "description": "Data on a specific change, represented by a pair of before and after resource snapshots.",
+      "type": "object",
+      "properties": {
+        "changeId": {
+          "description": "The change ID. Valid and unique within the specified resource only.",
+          "type": "string"
+        },
+        "beforeSnapshot": {
+          "description": "The snapshot before the change.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ResourceSnapshotData"
+            }
+          ]
+        },
+        "afterSnapshot": {
+          "description": "The snapshot after the change.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/ResourceSnapshotData"
+            }
+          ]
+        }
+      },
+      "required": [
+        "changeId",
+        "beforeSnapshot",
+        "afterSnapshot"
+      ]
+    },
+    "ResourceSnapshotData": {
+      "description": "Data on a specific resource snapshot.",
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "description": "The time when the snapshot was created.\nThe snapshot timestamp provides an approximation as to when a modification to a resource was detected.  There can be a difference between the actual modification time and the detection time.  This is due to differences in how operations that modify a resource are processed, versus how operation that record resource snapshots are processed.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "content": {
+          "description": "The resource snapshot content (in resourceChangeDetails response only).",
+          "type": "object"
+        }
+      },
+      "required": [
+        "timestamp"
+      ]
+    },
+    "ResourceChangeDetailsRequestParameters": {
+      "description": "The parameters for a specific change details request.",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Specifies the resource for a change details request.",
+          "type": "string"
+        },
+        "changeId": {
+          "description": "Specifies the change ID.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "resourceId",
+        "changeId"
+      ]
+    },
+    "DateTimeInterval": {
```

Comment:
startTime and endTime are more consistent property names for these values, when looking at other APIs where datetime filtering is used